ElectronClient/app/gui/NoteText.jsx

```diff
@@ -434,6 +448,10 @@ class NoteTextComponent extends React.Component {
 			await this.reloadNote(nextProps);
 		} else if ('noteId' in nextProps && nextProps.noteId !== this.props.noteId) {
 			await this.reloadNote(nextProps);
+		} else if ('noteTags' in nextProps && this.areNoteTagsModified(nextProps.noteTags, this.state.noteTags)) {
+			await this.setState({
```

Comment:
Any reason why there's an "await" here? I don't believe setState returns a promise so the "await" won't do anything.

Reply:
Hmm...not sure why I added this. Removed this piece of code.